### DISCUSSION TITLE
custom colors don't change once set even after clearing the cache

### DISCUSSION THREAD

**Discussion by CJ-Uy-EVC** [tailwindlabs/tailwindcss#11557] [Help]
I am currently working on a nuxt 3 project. After setting my custom colors the first time it no longer changes even when the value in the tailwind.config.js is changed. I've already tried deleting the `.nuxt` folder and running `npx nuxi cleanup` yet when restarting the page it still does not change anything. To add to the confusion going to the `Tailwind Viewer: http://localhost:3000/_tailwind/ ` shows that the NEW correct custom colors not the previous ones being shown in the available colors section. I'm really not quite sure how to fix this issue. Thank you for your time and help in advance!

tailwind.config.js
```
import FormKitVariants from "@formkit/themes/tailwindcss";

module.exports = {
    content: [
        "./components/**/*.{js,vue,ts}",
        "./layouts/**/*.vue",
        "./pages/**/*.vue",
        "./plugins/**/*.{js,ts}",
        "./nuxt.config.{js,ts}",
        "./app.vue",
        "./src/**/*.{html,js,vue}",
        "./node_modules/@formkit/themes/dist/tailwindcss/genesis/index.cjs",
    ],
    theme: {
        extend: {
            screens: {
                xs: "614px",
                sm: "1002px",
                md: "1022px",
                lg: "1092px",
                xl: "1280px",
            },
            colors: {
                primary: "#0083f7",
                secondary: "#22397d",
                offWhite: "#f8f9fe",
                important: "#d93558",
                clearGreen: "#5cb388",
            },
        },
    },
    plugins: [FormKitVariants],
};
```

On my component:
```
<button
    class="rounded bg-primary px-4 py-2 font-bold text-offWhite"
    @click="signIn('google')"
>
    Sign In With Google
</button>
```

I also found that if you do `class = "hover:bg-primary"` it uses the currently place correct color not the previous one

**Comment by CJ-Uy-EVC** [comment]
Could not find the REAL Solution so renaming it was the only option to keep the main name I just did:
`"primary": { 100: "#23397D", 200: "#1B2B65" },`. I've tried deleting the `node_modules` folder and clearing cache to no avail. WEIRDER YET, I've tried to delete the github folder and reclonning it yet it was still the same! I have literally no idea what is going on with this bug.

**Comment by adamwathan** [comment]
Hey! If you create a proper reproduction project (a small GitHub someone can clone down that demonstrates the problem) you're more likely to find someone who can take the time to look into it 👍
